test/helpers/WitnetRequestBoardNotCompliantTroyHorse.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "../../contracts/utils/Initializable.sol";
+import "../../contracts/utils/Proxiable.sol";
+
+/**
+ * @title Witnet Requests Board Troy Horse 1
+ * @notice Contract to test proxy upgrade assertions. 
+ * @dev This contract enables posting requests that Witnet bridges will insert into the Witnet network
+  * The result of the requests will be posted back to this contract by the bridge nodes too.
```

Comment:
I think we can replace these 2 lines with a description on how this contract is evil.

Reply:
TrojanHorse instead of TroyHorse :rofl:

Reply:
Renamed to `WitnetRequestBoardTrojanHorseNotUpgrdable` with description:
- *Upgrading an existing `WitnetRequestBoard` implementation with an instance of this kind (i.e. Proxiable but not-Upgradable), should not be permitted by the `WitnetProxy`. The contract has been created for testing purposes.*

Added `WitnetRequestBoardTrojanHorseBadProxiable`, and corresponding unitary test, with description:
- *Upgrading an existing WitnetRequestBoard implementation with an instance of this kind (i.e. Proxiable and Upgradable) but with a `proxiableUUID()` value different to the one required for WitnetRequestBoards, should not be permitted by the WitnetProxy. The contract has been created for testing purposes.*